README.md

```diff
@@ -13,34 +13,44 @@ This project is licensed under <a href="http://opensource.org/licenses/NPOSL-3.0
 https://github.com/jhundal/src/blob/master/bin/images/pvacseq-code.jpg
 
 ## Installation Instructions
-pVAC-Seq could either be used via cloning the git repo or by downloading the tarball.
-We strongly recommend the users to access it via git repo since updates are pushed before they make it to the stable release.
+pVAC-Seq could either be used via pip, cloning the git repo or by downloading the tarball.
+We strongly recommend the users to access it via pip as it will automatically fetch the most recent version, and automatically install the program for you
```

Comment:
We strongly recommend installation via pip as it will automatically fetch the most recent version and install the program for you.